Task 9/features/steps/login.refs.js

```diff
@@ -0,0 +1,102 @@
+const { By, Key, until } = require('selenium-webdriver');
+const BasePage = require('./basepage')
+
+
+
+class LoginPage extends BasePage{
+
+    goToUrl(theURL){
+        this.driver.get(theURL);
+    }
+
+    async headerLogoClickOn(){
+        return await this.driver.findElement(By.className("navbar-brand logo-link")).click();
+    }
+
+    async footerLogoClickOn(){
+        await this.driver.findElement(By.css('a[href="https://kashkick.com"]')).click();
+    }
+
+    async getLoginPageText(){
+       return await this.driver.findElement(By.xpath('//*[(contains(text(), "Greatest Reward"))]')).getText();
+    }
+
+    async getCurrentPageUrl(){
+       return await this.driver.getCurrentUrl();
+    }
+
+    async footerLogoClickOn(){
+        return await this.driver.findElement(By.className("navbar-brand logo-link")).click();
+    }
+    
+    async forgotPsswrdBtnClickOn(){
+        return await this.driver.findElement(By.className('hoverLight')).click();
+    }
+
+    async emailFillIn(email){
+        await this.driver.findElement(By.name("Email")).sendKeys(email);
+    }
+
+    async psswrdFillIn(password){
+        await this.driver.findElement(By.name("Password")).sendKeys(password);
+    }
+
+    async requestPsswrdBtnClickOn(){
+        await this.driver.findElement(By.className('forgot-password__btn')).click()
+    }
+
+    async signInBtnClickOn(){
+        await this.driver.findElement(By.name('Login')).click();
+    }
+
+    async getSuccessPsswrdRequestText(){
+        return await this.driver.findElement(By.xpath('//*[(contains(text(), "Please look for our"))]')).getText();
```

Comment:
bad locator, find a locator that will not use text, and combine all the methods that get text from that alert